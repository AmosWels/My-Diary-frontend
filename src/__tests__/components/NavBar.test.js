import React from "react";
import { shallow, configure } from "enzyme";
import NavBar from "../../navigation/navBar";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });

describe("SignupUser component", () => {
  const props = {
    isOpen: false,
    toggle: jest.fn(),
    setState: jest.fn(),
    clearToken: jest.fn()
  };
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<NavBar {...props} />);
  });
  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("toggle should be called", () => {
    const mySpy = jest.spyOn(wrapper.instance(), "toggle");
    wrapper.find("NavbarToggler").simulate("click");
    expect(mySpy).toHaveBeenCalledTimes(0);
  });
});
