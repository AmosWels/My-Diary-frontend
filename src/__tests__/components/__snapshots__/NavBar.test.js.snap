// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupUser component should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavBar
    isOpen={false}
    setState={[MockFunction]}
    toggle={[MockFunction]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Navbar
        expand="md"
        id="mynav"
        light={true}
        tag="nav"
      >
        <NavbarBrand
          href="/view-articles"
          id="navbar-brand"
          tag="a"
        >
          My Diary
        </NavbarBrand>
        <NavbarToggler
          onClick={[Function]}
          tag="button"
          type="button"
        />
        <Collapse
          appear={false}
          enter={true}
          exit={true}
          in={false}
          isOpen={false}
          mountOnEnter={false}
          navbar={true}
          onEnter={[Function]}
          onEntered={[Function]}
          onEntering={[Function]}
          onExit={[Function]}
          onExited={[Function]}
          onExiting={[Function]}
          tag="div"
          timeout={350}
          unmountOnExit={false}
        >
          <Nav
            className="ml-auto"
            navbar={true}
            tag="ul"
            vertical={false}
          >
            <ul
              className="navbar-nav"
            >
              <li
                className="nav-item active"
              >
                <a
                  className="nav-link"
                  href="/"
                  id="navv-link"
                >
                  Welcome To Your Diary
                </a>
              </li>
              <li
                className="nav-item active"
              >
                <a
                  className="nav-link"
                  href="/"
                  id="navv-link"
                >
                  Profile
                </a>
              </li>
              <li
                className="nav-item active"
              >
                <a
                  className="nav-link"
                  href="/"
                  id="navv-link"
                >
                  Logout
                </a>
              </li>
              <li
                className="nav-item active"
              >
                <img
                  alt="user"
                  id="profile"
                  src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                />
              </li>
              <li
                className="nav-item active"
              >
                <a
                  className="nav-link"
                  href="/entries"
                />
              </li>
            </ul>
          </Nav>
        </Collapse>
      </Navbar>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <NavbarBrand
            href="/view-articles"
            id="navbar-brand"
            tag="a"
          >
            My Diary
          </NavbarBrand>,
          <NavbarToggler
            onClick={[Function]}
            tag="button"
            type="button"
          />,
          <Collapse
            appear={false}
            enter={true}
            exit={true}
            in={false}
            isOpen={false}
            mountOnEnter={false}
            navbar={true}
            onEnter={[Function]}
            onEntered={[Function]}
            onEntering={[Function]}
            onExit={[Function]}
            onExited={[Function]}
            onExiting={[Function]}
            tag="div"
            timeout={350}
            unmountOnExit={false}
          >
            <Nav
              className="ml-auto"
              navbar={true}
              tag="ul"
              vertical={false}
            >
              <ul
                className="navbar-nav"
              >
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Welcome To Your Diary
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Profile
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Logout
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <img
                    alt="user"
                    id="profile"
                    src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                  />
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/entries"
                  />
                </li>
              </ul>
            </Nav>
          </Collapse>,
        ],
        "expand": "md",
        "id": "mynav",
        "light": true,
        "tag": "nav",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": "My Diary",
            "href": "/view-articles",
            "id": "navbar-brand",
            "tag": "a",
          },
          "ref": null,
          "rendered": "My Diary",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "onClick": [Function],
            "tag": "button",
            "type": "button",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "appear": false,
            "children": <Nav
              className="ml-auto"
              navbar={true}
              tag="ul"
              vertical={false}
            >
              <ul
                className="navbar-nav"
              >
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Welcome To Your Diary
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Profile
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Logout
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <img
                    alt="user"
                    id="profile"
                    src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                  />
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/entries"
                  />
                </li>
              </ul>
            </Nav>,
            "enter": true,
            "exit": true,
            "in": false,
            "isOpen": false,
            "mountOnEnter": false,
            "navbar": true,
            "onEnter": [Function],
            "onEntered": [Function],
            "onEntering": [Function],
            "onExit": [Function],
            "onExited": [Function],
            "onExiting": [Function],
            "tag": "div",
            "timeout": 350,
            "unmountOnExit": false,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <ul
                className="navbar-nav"
              >
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Welcome To Your Diary
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Profile
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Logout
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <img
                    alt="user"
                    id="profile"
                    src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                  />
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/entries"
                  />
                </li>
              </ul>,
              "className": "ml-auto",
              "navbar": true,
              "tag": "ul",
              "vertical": false,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Welcome To Your Diary
                    </a>
                  </li>,
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Profile
                    </a>
                  </li>,
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Logout
                    </a>
                  </li>,
                  <li
                    className="nav-item active"
                  >
                    <img
                      alt="user"
                      id="profile"
                      src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                    />
                  </li>,
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/entries"
                    />
                  </li>,
                ],
                "className": "navbar-nav",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Welcome To Your Diary
                    </a>,
                    "className": "nav-item active",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Welcome To Your Diary",
                      "className": "nav-link",
                      "href": "/",
                      "id": "navv-link",
                    },
                    "ref": null,
                    "rendered": "Welcome To Your Diary",
                    "type": "a",
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Profile
                    </a>,
                    "className": "nav-item active",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Profile",
                      "className": "nav-link",
                      "href": "/",
                      "id": "navv-link",
                    },
                    "ref": null,
                    "rendered": "Profile",
                    "type": "a",
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Logout
                    </a>,
                    "className": "nav-item active",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Logout",
                      "className": "nav-link",
                      "href": "/",
                      "id": "navv-link",
                    },
                    "ref": null,
                    "rendered": "Logout",
                    "type": "a",
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <img
                      alt="user"
                      id="profile"
                      src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                    />,
                    "className": "nav-item active",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "user",
                      "id": "profile",
                      "src": "https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      className="nav-link"
                      href="/entries"
                    />,
                    "className": "nav-item active",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "className": "nav-link",
                      "href": "/entries",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "a",
                  },
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Navbar
          expand="md"
          id="mynav"
          light={true}
          tag="nav"
        >
          <NavbarBrand
            href="/view-articles"
            id="navbar-brand"
            tag="a"
          >
            My Diary
          </NavbarBrand>
          <NavbarToggler
            onClick={[Function]}
            tag="button"
            type="button"
          />
          <Collapse
            appear={false}
            enter={true}
            exit={true}
            in={false}
            isOpen={false}
            mountOnEnter={false}
            navbar={true}
            onEnter={[Function]}
            onEntered={[Function]}
            onEntering={[Function]}
            onExit={[Function]}
            onExited={[Function]}
            onExiting={[Function]}
            tag="div"
            timeout={350}
            unmountOnExit={false}
          >
            <Nav
              className="ml-auto"
              navbar={true}
              tag="ul"
              vertical={false}
            >
              <ul
                className="navbar-nav"
              >
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Welcome To Your Diary
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Profile
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/"
                    id="navv-link"
                  >
                    Logout
                  </a>
                </li>
                <li
                  className="nav-item active"
                >
                  <img
                    alt="user"
                    id="profile"
                    src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                  />
                </li>
                <li
                  className="nav-item active"
                >
                  <a
                    className="nav-link"
                    href="/entries"
                  />
                </li>
              </ul>
            </Nav>
          </Collapse>
        </Navbar>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <NavbarBrand
              href="/view-articles"
              id="navbar-brand"
              tag="a"
            >
              My Diary
            </NavbarBrand>,
            <NavbarToggler
              onClick={[Function]}
              tag="button"
              type="button"
            />,
            <Collapse
              appear={false}
              enter={true}
              exit={true}
              in={false}
              isOpen={false}
              mountOnEnter={false}
              navbar={true}
              onEnter={[Function]}
              onEntered={[Function]}
              onEntering={[Function]}
              onExit={[Function]}
              onExited={[Function]}
              onExiting={[Function]}
              tag="div"
              timeout={350}
              unmountOnExit={false}
            >
              <Nav
                className="ml-auto"
                navbar={true}
                tag="ul"
                vertical={false}
              >
                <ul
                  className="navbar-nav"
                >
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Welcome To Your Diary
                    </a>
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Profile
                    </a>
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Logout
                    </a>
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <img
                      alt="user"
                      id="profile"
                      src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                    />
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/entries"
                    />
                  </li>
                </ul>
              </Nav>
            </Collapse>,
          ],
          "expand": "md",
          "id": "mynav",
          "light": true,
          "tag": "nav",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": "My Diary",
              "href": "/view-articles",
              "id": "navbar-brand",
              "tag": "a",
            },
            "ref": null,
            "rendered": "My Diary",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "onClick": [Function],
              "tag": "button",
              "type": "button",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "appear": false,
              "children": <Nav
                className="ml-auto"
                navbar={true}
                tag="ul"
                vertical={false}
              >
                <ul
                  className="navbar-nav"
                >
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Welcome To Your Diary
                    </a>
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Profile
                    </a>
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Logout
                    </a>
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <img
                      alt="user"
                      id="profile"
                      src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                    />
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/entries"
                    />
                  </li>
                </ul>
              </Nav>,
              "enter": true,
              "exit": true,
              "in": false,
              "isOpen": false,
              "mountOnEnter": false,
              "navbar": true,
              "onEnter": [Function],
              "onEntered": [Function],
              "onEntering": [Function],
              "onExit": [Function],
              "onExited": [Function],
              "onExiting": [Function],
              "tag": "div",
              "timeout": 350,
              "unmountOnExit": false,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": <ul
                  className="navbar-nav"
                >
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Welcome To Your Diary
                    </a>
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Profile
                    </a>
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/"
                      id="navv-link"
                    >
                      Logout
                    </a>
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <img
                      alt="user"
                      id="profile"
                      src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                    />
                  </li>
                  <li
                    className="nav-item active"
                  >
                    <a
                      className="nav-link"
                      href="/entries"
                    />
                  </li>
                </ul>,
                "className": "ml-auto",
                "navbar": true,
                "tag": "ul",
                "vertical": false,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <li
                      className="nav-item active"
                    >
                      <a
                        className="nav-link"
                        href="/"
                        id="navv-link"
                      >
                        Welcome To Your Diary
                      </a>
                    </li>,
                    <li
                      className="nav-item active"
                    >
                      <a
                        className="nav-link"
                        href="/"
                        id="navv-link"
                      >
                        Profile
                      </a>
                    </li>,
                    <li
                      className="nav-item active"
                    >
                      <a
                        className="nav-link"
                        href="/"
                        id="navv-link"
                      >
                        Logout
                      </a>
                    </li>,
                    <li
                      className="nav-item active"
                    >
                      <img
                        alt="user"
                        id="profile"
                        src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                      />
                    </li>,
                    <li
                      className="nav-item active"
                    >
                      <a
                        className="nav-link"
                        href="/entries"
                      />
                    </li>,
                  ],
                  "className": "navbar-nav",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        className="nav-link"
                        href="/"
                        id="navv-link"
                      >
                        Welcome To Your Diary
                      </a>,
                      "className": "nav-item active",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Welcome To Your Diary",
                        "className": "nav-link",
                        "href": "/",
                        "id": "navv-link",
                      },
                      "ref": null,
                      "rendered": "Welcome To Your Diary",
                      "type": "a",
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        className="nav-link"
                        href="/"
                        id="navv-link"
                      >
                        Profile
                      </a>,
                      "className": "nav-item active",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Profile",
                        "className": "nav-link",
                        "href": "/",
                        "id": "navv-link",
                      },
                      "ref": null,
                      "rendered": "Profile",
                      "type": "a",
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        className="nav-link"
                        href="/"
                        id="navv-link"
                      >
                        Logout
                      </a>,
                      "className": "nav-item active",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Logout",
                        "className": "nav-link",
                        "href": "/",
                        "id": "navv-link",
                      },
                      "ref": null,
                      "rendered": "Logout",
                      "type": "a",
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <img
                        alt="user"
                        id="profile"
                        src="https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1"
                      />,
                      "className": "nav-item active",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "alt": "user",
                        "id": "profile",
                        "src": "https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?fit=256%2C256&quality=100&ssl=1",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "img",
                    },
                    "type": "li",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <a
                        className="nav-link"
                        href="/entries"
                      />,
                      "className": "nav-item active",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "className": "nav-link",
                        "href": "/entries",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "a",
                    },
                    "type": "li",
                  },
                ],
                "type": "ul",
              },
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
